app/admin/dashboard.rb

```diff
@@ -1,33 +0,0 @@
-ActiveAdmin.register_page "Dashboard" do
```

Comment:
Decided the dashboard is out of scope, esp. since it's not in the original design document.